src/server/handlers/github.js

```diff
@@ -150,10 +152,9 @@ export const listRepositories = async (req, res) => {
 
   const body = {
     status: 'success',
-    payload: {
-      code: 'github-list-repositories',
-      repos: response.body
-    }
+    code: 'github-list-repositories',
+    // XXX drop response specific prop, make response a little more general purpose
+    ...normalize(response.body, repoList)
```

Comment:
I don't find it confusing, but unnecessary for sure, especially when the actions blindly pass whole response to reducer which results in `action.payload.payload` type of calls.

But we use this pattern for all of our responses from BSI APIs, which means a lot of places to update if we decide to switch to new scheme and very confusing 'in-between' state in the meantime.